xml/System/Span`1.xml

```diff
@@ -155,8 +155,8 @@ If `array` is `null`, this constructor returns a `null` `Span<T>`.
       </Parameters>
       <Docs>
         <param name="pointer">A pointer to the starting address of a specified number of bytes in memory.</param>
```

Comment:
Can you fix this as well? It states bytes still.

Reply:
Will do.